### PR TITLE
feat(cli,docs): add sharedspaces send and agent notification skill

### PR DESCRIPTION
**Pull request by maraf**
## Why

When a user kicks off long-running agent work and walks away from the terminal, they have no way to learn about milestones, blockers, or completion. SharedSpaces already pushes items to joined members in real time (including web push to phones), so it is a natural out-of-band notification channel.

The blocker was on the CLI side: the server already accepts `contentType: "text"` items with a TTL, but the CLI could only upload files. There was no ergonomic way to post a short status message.

## What changed

**New `sharedspaces send` command.** Posts a short-lived text item to a joined space.

```bash
sharedspaces send "Build finished" --space-id <guid> --ttl 3600
echo "Long update" | sharedspaces send --space-id <guid>
```

- `SharedSpacesApiClient.SendTextAsync(...)` PUTs to `/v1/spaces/{id}/items/{guid}` with `contentType=text`, `content`, and optional `ttlSeconds`
- Message comes from an argument, or from stdin when omitted or passed as `-`
- `--json` for deterministic parsing by agents; error handling and exit-code behavior mirror `upload`

**New `sharedspaces-notify` skill** (`.github/skills/`). Teaches agents when and how to notify.

- Opt-in and user-initiated. The description explicitly excludes normal interactive work where the user is watching the terminal, so it should not fire on its own.
- Setup runs once: verify the CLI, then `sharedspaces spaces --json`. Zero spaces means asking the user for a join string; one means use it; more than one means asking which. It never guesses.
- Derives a short work label (`<repo>/<branch>`) so messages are attributable when several agent sessions share a space.
- Sends text rather than files, default TTL 3600s, format `[<workLabel>] <STATUS> - <summary>` with `MILESTONE` / `DONE` / `BLOCKED` / `NEEDS INPUT` / `FAILED`.
- Explicit anti-flood rules, plus a rule that a failed send never blocks the actual work.

## Notes for reviewers

- Persisted setup state is `{ spaceId, workLabel }` written to the session artifacts directory, never the repo. `serverUrl` and the JWT are deliberately not duplicated there since the CLI already resolves them from `~/.sharedspaces/config.json`.
- The `send` error paths print to stderr but exit `0`. That is pre-existing behavior shared with `upload`, so I left it alone rather than changing exit-code semantics for one command. Happy to fix it repo-wide in a follow-up if you want.
- No UI changes, so the screenshot workflow does not apply.

## Testing

8 new unit tests covering URL and auth headers, multipart payload shape, TTL present/absent, response parsing, and error mapping. Full suite: 329 passed, 0 failed. Also smoke-tested `send --help` and the empty-message, invalid-TTL, and unknown-space paths.

### CODE REVIEW

File: tests/SharedSpaces.Cli.Core.Tests/SendTextTests.cs [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    public async Task SendTextAsync_SendsTextContentTypeAndContent()
+    {
+        var (client, handler) = CreateClient();
+        using var _ = client;
+
+        await client.SendTextAsync(ServerUrl, SpaceId, ItemId, Token, "Build finished");
+
+        handler.Body.Should().Contain("name=id").And.Contain(ItemId);
+        handler.Body.Should().Contain("name=contentType").And.Contain("text");
+        handler.Body.Should().Contain("name=content").And.Contain("Build finished");
```
[the remark this answers is not in the document]
Comment (maraf) [reply tests/SharedSpaces.Cli.Core.Tests/SendTextTests.cs]: Accepted - fixed in 619e791. The test handler now iterates the `MultipartFormDataContent` parts and builds a field name -> value dictionary (trimming quotes off `ContentDisposition.Name`), so the assertions check exact values like `contentType` == `text` instead of substring-matching the serialized payload.

This also fixes a latent false-negative you did not mention: the old `NotContain("ttlSeconds")` check would have spuriously failed if a message body ever contained that word, whereas `NotContainKey("ttlSeconds")` tests the actual intent. All 69 tests in the project still pass.
